osu.Game.Tests/Visual/Background/TestCaseBackgroundScreenBeatmap.cs

```diff
@@ -275,6 +275,8 @@ private void setupUserSettings()
 
         private class DummySongSelect : PlaySongSelect
         {
+            protected override UserStatusOnline InitialScreenStatus => null;
```

Comment:
why are these overrides required? (as in, they shouldn't be)

Reply:
I overrided these in test classes because they seem to break unit tests.

Reply:
No doubt. I'm saying that's not the correct fix.